spec/acceptance/schema_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'spec_helper_acceptance'
+
+describe 'openldap::server::schema' do
+
+  context 'without parameters' do
+    it 'creates an overlay' do
+      pp = <<-EOS
+      class { 'openldap::server': }
+      ->
+      openldap::server::schema { 'core':
+        ensure => present,
+      }
+      EOS
+
+      apply_manifest(pp, :catch_failures => true)
+      #apply_manifest(pp, :catch_changes => true)
```

Comment:
I have been unable to get the test suite to work, I have been testing it manually in a new vps. I started with the overlay_spec.rb file which also has the second test commented. I'll enable it in schema_spec.rb